ansible-tests/validations/instackenv.yaml

```diff
@@ -0,0 +1,18 @@
+---
+- hosts: undercloud
+  become: false
```

Comment:
Same comment about 'become', where I'd like this to be the global default.
If it is the case, we don't need to specify it here.